include/boost/geometry/algorithms/detail/is_valid/has_invalid_coordinate.hpp

```diff
@@ -80,26 +80,26 @@ struct indexed_has_invalid_coordinate
 
 struct range_has_invalid_coordinate
 {
+private:
     struct point_has_valid_coordinates
     {
         template <typename Point>
-        static inline bool apply(Point const& point)
+        inline bool operator()(Point const& point)
         {
-            return ! point_has_invalid_coordinate::apply(point);
+            return point_has_invalid_coordinate::apply(point);
```

Comment:
This predicate is no longer `point_has_valid_coordinates`. I haven't checked but since there already is `point_has_invalid_coordinate` it's possible that this one is not needed.

Reply:
I changed it to lambda and removed `point_has_valid_coordinates`